tests/test_eload_utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+import unittest
+
+from eva_submission import ROOT_DIR
+from eva_submission.eload_utils import backup_file
+
+
+def touch(f):
+    open(f, 'w').close()
+
+
+class TestEloadUtils(unittest.TestCase):
+    resources_folder = os.path.join(ROOT_DIR, 'tests', 'resources')
+
+    def tearDown(self) -> None:
+        test = os.path.join(self.resources_folder, 'testbackupfile.txt')
+        for i in range(5):
+            os.remove(test + '.' + str(i))
+
+    def test_backup_file(self):
```

Comment:
If my other comment is correct, I think this test isn't quite right either... Can we test this by calling `backup_file(test)` 5 times and checking whether the right files exist?

Reply:
One thing this made me realised the name `backup` is not really appropriate since I'm not copying the file but renaming it. 
If I call backup_file(test) I need to recreate the file to backup every time.

Reply:
Right I see... in theory it could really backup the file though, right?  Since in the actual code you clear the config so it should overwrite it anyway.